app/controllers/photos_controller.rb

```diff
@@ -5,7 +5,10 @@ class PhotosController < ApplicationController
   # GET /photos
   # GET /photos.json
   def index
-    @photos = @album.photos
+    @desc = true
+    @desc = false if params[:desc] == 'true'
```

Comment:
Oh I see, you are using it in the view. In this case you can probably simplify the order_by method by just passing the @desc variable to it instead of checking params again.